client/my-sites/upgrades/cart/cart-item.jsx

```diff
@@ -99,7 +144,8 @@ module.exports = React.createClass( {
 					<span className="product-price">
 						{ this.price() }
 					</span>
-					{ this.removeButton() }
+					{ this.domainVolumeSelection() }
```

Comment:
I think it would be nice to inline this here with some helper variables at the top of `render()`

```js
render() {
	const { cartItem } = this.props;
	const volumeSelectionIsEnabled = config.isEnabled( 'upgrades/cart/multi-year-domain' );
	const showVolumeSelection =
		volumeSelectionIsEnabled &&
		( this.isBundlePlanApplied() || cartItem.free_trial || isDomainRegistering( cartItem ) );

	…

	</span>
	{ showVolumeSelection &&
		<select name="product-domain-volume"
			className="product-domain-volume"
			…>
			…
		</select>
	}
}
```